sdk/va-report-components/documentation/docs/api/SASReportElement.md

```diff
@@ -52,6 +52,12 @@ than one section.
 
 default value: `'auto'`
 
+### `restrictViewportGestures: boolean`
+
+If `true`, some interactive viewport features may be restricted. Currently the only restriction is that a modifier key is required to enable zooming with the scroll wheel. When the user engages the scroll wheel a transluecent overlay is placed over the visual with a message indicating which modifier key is required to enable zooming. The overlay disappears when the key is pressed.
```

Comment:
I don't think we should use phrases like "may be  restricted", or words like "currently".  We should say, in a straightforward way, exactly what the property controls, in terms of what the effect is on the user.  If that implementation changes in the future to include more gesture restrictions, then that gets noted at that time. And we probably don't need to explain all the nuances of the implementation, like how an overlay is shown.  That is a detail that could easily change without requiring deprecation or change in api. I think that the key thing to explain is the forced use of a modifier key to zoom, and then probably explaining to the user why that might be desired.

```suggestion
When `true`, report objects that support zooming require a modifier key be used in addition to the scroll-wheel. This should be used when embedding elements in a layout that can cause overflow, which will allow the scroll-wheel action to be reserved for page scrolling.
```

Reply:
I like Brad's suggestion. The first occurrence of scroll wheel doesn't need a hyphen. (Keep the hyphen in second occurrence.)

To help non-native English speakers, we should break the long second sentence into two sentences.  "Use the modifier key when embedding elements in a layout that causes overflow. This reserves the scroll-wheel action for page scrolling."

Reply:
I think "Use the modifier key" doesn't make sense.  We are advising the developer to use the api option that would then require the user to use the modifier key.

So maybe "Enable `restrictViewportGestures` when embedding elements in a layout that causes overflow.  This reserves the scroll-wheel action for page scrolling."

Reply:
Thanks! I have made the suggested changes.

Reply:
Oops! I missed @brmorr last comment.
I can understand "Use the modifier key" in that it is implicit that setting the api means you will use the modifier key, but I agree with Brad that it didn't read to me quite right initially. His latest suggestion makes more sense to me.

Reply:
Yes, use Brad's latest suggestion. :)